packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/Characteristics/Characteristics.spec.js

```diff
@@ -0,0 +1,55 @@
+import { mount } from 'enzyme'
+import { Grommet } from 'grommet'
+import React from 'react'
+import sinon from 'sinon'
+import zooTheme from '@zooniverse/grommet-theme'
+
+import { task as mockTask } from '@plugins/tasks/SurveyTask/mock-data'
+import Characteristics from './Characteristics'
+import CharacteristicSection from './components/CharacteristicSection'
+
+describe('Component > Characteristics', function () {
+  let wrapper, onFilterSpy
+
+  before(function () {
+    onFilterSpy = sinon.spy()
+    wrapper = mount(
+      <Characteristics
+        characteristics={mockTask.characteristics}
+        characteristicsOrder={mockTask.characteristicsOrder}
+        images={mockTask.images}
+        onFilter={onFilterSpy}
+      />, {
+        wrappingComponent: Grommet,
+        wrappingComponentProps: { theme: zooTheme }
+      }
```

Comment:
This can be removed.
```suggestion
      />
```